cfgmgr/buffermgrdyn.cpp

```diff
@@ -711,6 +720,43 @@ bool BufferMgrDynamic::isHeadroomResourceValid(const string &port, const buffer_
     return result;
 }
 
+task_process_status BufferMgrDynamic::removeAllPGsFromPort(const string &port)
```

Comment:
sometimes functions are name with PGs, pg and PriorityGroups. Can you choose one way and have the names aligned?

Reply:
Let's use `pg` as it's a common term in buffer management and has already been widely used in this file.
Two functions need to be renamed:
`removeAllPGsFromPort` => `removeAllPgsFromPort`
`refreshPriorityGroupsForPort` => `refreshPgsForPort`